pynest/nest/__init__.py

```diff
@@ -178,8 +173,12 @@ def test():
 
     hl_api.set_debug(debug)
 
+from .pynestkernel import *
+from lib.hl_api_helper import *
+
+for name in os.listdir(os.path.join(os.path.dirname(__file__), "lib")):
+    if name.endswith(".py") and not name.startswith('__'):
+        exec("from lib.{0} import *".format(name[:-3]))
```

Comment:
I am not quite sure what is going on here (maybe I am just too tired) but what kind of files of name `lib.xyz.py` are we expecting here?